waspc/cli/src/Wasp/Cli/Command/Uninstall.hs

```diff
@@ -0,0 +1,52 @@
+module Wasp.Cli.Command.Uninstall
+  ( uninstall,
+  )
+where
+
+import Control.Monad (when)
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (fromAbsDir, fromAbsFile, (</>))
+import System.Exit (die)
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.FileSystem (deleteDirectoryIfExists, deleteFileIfExists, getHomeDir, getUserCacheDirPath, waspExecutableInHomeDir, waspInstallationDirInHomeDir)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import Wasp.Cli.Command.Telemetry.Common (getWaspCacheDirPath)
+import qualified Wasp.Message as Msg
+
+-- | Removes Wasp CLI from the system.
+-- It removes the follwing:
+-- {home}/.local/share/wasp-lang
+-- {home}/.local/bin/wasp
+-- {home}/.cache/wasp
+uninstall :: Command ()
+uninstall = do
+  cliSendMessageC $ Msg.Info "Uninstalling Wasp ..."
+  liftIO removeWaspFiles
+  cliSendMessageC $ Msg.Success "Uninstalled Wasp"
+
+removeWaspFiles :: IO ()
+removeWaspFiles = do
+  homeDir <- getHomeDir
+  userCacheDir <- getUserCacheDirPath
```

Comment:
Seeing some inconsistency here: one function has `Path` at the end, the other does not. I am not sure what is better, but let's stick to one approach, at least here.

Reply:
Great catch, I'll remove the suffix 👍